src/PythonClient/src/quixstreams/dataframes/models/rows.py

```diff
@@ -21,3 +20,15 @@ class Row:
     headers: Optional[Mapping | MessageHeaders] = None
     latency: Optional[float] = None
     leader_epoch: Optional[int] = None
+
+    def __getitem__(self, item):
+        return self.value[item]
+
+    def __bool__(self):
+        return bool(self.value)
```

Comment:
What is the purpose of having `__bool__` implemented on Row object? 
Plus it checks `bool(self.value)` which can be `False` if value is any of `{}, [], 0, ""`, which are valid values.